src/test/ui/consts/const-eval/issue-100878.rs

```diff
@@ -0,0 +1,11 @@
+// This checks that the const-eval ICE in issue #100878 does not recur.
+//
+// build-pass
+
+pub fn bitshift_data(data: [u8; 1]) -> u8 {
+    data[0] << 8
+}
+
+fn main() {
```

Comment:
Would it be better to condense this to 1 line, since there is nothing in `main`?

```rust
fn main() {}
```

Note: Not sure if that changes the test semantics. If so, please ignore this comment.

Reply:
Sure, done!